RTN-004.tex

```diff
@@ -188,54 +221,40 @@ \subsection{Delegate Groups}\label{ssec:sel_grps}
-\centering
-\caption{Approximate targeted distribution of selection committee representation.}\label{tab:selection_committee}
-\begin{tabular}{ll}
-\hline
-Number & Source \\
-\hline \hline
-4 & Science Advisory Council (SAC) and the LSST Science Collaborations \\
-3 & Community Engagement Team and Rubin Observatory Director's Office \\
-3 & Rubin Operations (e.g., Data Production, SPaRE, RSP, In-Kind Program Coordinators) \\
-2 & NOIR Lab (e.g., Community Science Data Center), DOE, or other institutions \\
-2 & Chilean astronomical community \\
-\hline
-\end{tabular}
-\end{table}
+\subsection{Selection Committee}\label{ssec:sel_comm}
 
-To keep committee participation manageable, committee members will be primarily tasked with providing shortlists of applicants for one or more groups.
-For example, committee members from the SAC and Science Collaborations would be primarily tasked with generating a shortlist for Group H, but would also provide input on the other groups.
-The CET and the Rubin directorate will merge these shortlists into a final list of DP0 delegates.
+The delegate selection committee will be comprised of members of the Rubin Observatory Community Engagement Team (CET) and the Rubin Observatory Director's Office.
+The selection committee will use the diversity-based selection criteria and process described in Section \ref{ssec:sel_crit}.
 
-All selection committee members will use the diversity-based selection criteria in Section \ref{ssec:sel_crit} when forming shortlists for the various groups.
-Prior to meeting, the CET will facilitate a session on unconscious bias for the selection committee members.
-The capacity to view form results without names will be provided to the committee.
+The selection committee will continue to seek advisement from the Rubin Observatory Science Advisory Council (SAC) and the LSST Science Collaborations by providing to them a report which summarizes the DP0 delegates identified for DP0.1, including distributions of DP0 delegates across the defined groups (Table \ref{tab:delegate_groups}) and diversity aspects (Sections \ref{ssec:sel_form}).
 
 \subsection{Selection Criteria}\label{ssec:sel_crit}
 
 Diversity in representation is the top priority.
-The groups described in Section \ref{ssec:sel_grps} are designed to achieve a diverse population of DP0 delegates in several facets: experience, expertise, career level, institution type, geographical location, and scientific field.
+The groups described in Section \ref{ssec:sel_grps} are designed to achieve a diverse population of DP0 delegates in several facets, such as experience, expertise, career level, institution type, geographical location, scientific field, and minority representation.
 The selection committee will also consider these aspects of diversity within the groups.
 
+The selection committee will use of diversity optimization software such as {\tt entrofy} \citep{2019arXiv190503314H} to inform process of identifying DP0 delegates.
```

Comment:
will make use of diversity optimization software such as

Reply:
fixed